### PR TITLE
Referrals Page with some styles

### PR DESCRIPTION
**Pull request by youngkidwarrior**

### CODE REVIEW

**Comment by youngkidwarrior** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #77** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/77?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈
    * **PR #78** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/78?utm_source=stack-comment-icon" target="_blank">Graphite</a> 

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/77?utm_source=stack-comment).

File: packages/app/features/referrals/screen.tsx
Comment (0xBigBoss) [inline packages/app/features/referrals/screen.tsx:1]: add a jest snapshot test please.

**Comment by youngkidwarrior** [comment]
### Merge activity

* **Jan 24, 10:08 PM**: [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/77) rebased this pull request as part of a merge.

**Review by 0xBigBoss — DISMISSED** [review]
It's a good start.

**Review by 0xBigBoss — APPROVED** [review]

**Pull request merged**